docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+networks:
+  mojaloop-net:
+    name: mojaloop-net
+
+
+services:
+  mifosCoreConnector:
+    image: mojaloop/mifos-core-connector:latest
+    build:
+      context: ../..
```

Comment:
it's a bit not clear for me, what should we have inside this `context-folder` (seems, it's not in scope of this git repo). So what should I do/have to be able to build `mifosCoreConnector` image on my local machine?

Reply:
I am going to check this out.  Seems something is wrong.